src/js/discharge-wizard/containers/GuidancePage.jsx

```diff
@@ -9,11 +9,16 @@ import { venueWarning } from '../config';
 
 class GuidancePage extends React.Component {
   componentDidMount() {
+    const prevScript = document.getElementById('uswds');
+    if (prevScript) { prevScript.remove(); }
+
     const script = document.createElement('script');
     script.type = 'text/javascript';
     script.async = true;
-    script.src = 'https://standards.usa.gov/assets/js/vendor/uswds.min.js';
+    script.id = 'uswds';
+    script.src = 'https://standards.usa.gov/assets/js/vendor/uswds.min.js?guidance';
```

Comment:
Looks like this is to reload the script so that some widget works, but why is the `?guidance` query needed?

Reply:
@U-DON I'm trying to get the USWDS script to reinitialize for the accordions here... looks like this doesn't work. Might need to make our component here to handle this...